### PR TITLE
Added LoginStatus requestSteamGuard()

### PR DESCRIPTION
**Pull request by WalkingCore**
LoginStatus requestSteamGuard() is a function that sends a request to the steam servers so the user can get their code in email and then use it in the actual login function.

### CODE REVIEW

File: SteamAPISession.cs [hunk 1]
```diff
@@ -145,6 +145,23 @@ public class ServerInfo {
             public DateTime serverTime;
             public String serverTimeString;
         }
+		
+		 /// <summary>
```
Comment (Overv) [inline SteamAPISession.cs]: Could you fix the indentation?

File: SteamAPISession.cs [hunk 2]
```diff
@@ -145,6 +145,23 @@ public class ServerInfo {
...
+        /// Sends the Steam servers a pre-request so you can recive your Steam Guard code through email.
+        /// </summary>
+        /// <param name="username">Username</param>
+        /// <param name="password">Password</param>
+        /// <returns>1 for success and 0 for failure</returns>
+        public LoginStatus steamGuardRequest(String username, String password) {
+            String response = steamRequest("ISteamOAuth2/GetTokenWithCredentials/v0001",
+                "client_id=DE45CD61&grant_type=password&username=" + Uri.EscapeDataString(username) + "&password=" + Uri.EscapeDataString(password) + "&x_emailauthcode=0&scope=read_profile%20write_profile%20read_client%20write_client");
+
+            if (((string) data["x_errorcode"]).Equals("steamguard_code_required"))
```
Comment (Overv) [inline SteamAPISession.cs]: Please use curly braces here

Comment (WalkingCore) [inline SteamAPISession.cs]: On what lines exactly?

**Comment by WalkingCore** [comment]
I don't see anything wrong with the indentation but sure I will put spaces.

**Comment by Overv** [comment]
You don't see that the indentation of the `else` line is completely off here?

https://github.com/WalkingCore/SteamWebAPI/commit/257472de566f011b0b0340e3776154fa5a863efe

**Comment by WalkingCore** [comment]
I don't think intendation affects the code...It should run just fine. :)

**Comment by Overv** [comment]
What is up with the broken code in `steamGuardRequest`?

**Comment by ErikHumphrey** [comment]
++

**Pull request merged**